lib/eldap/src/eldap.erl

```diff
@@ -384,6 +442,11 @@ Delete an entry.
 ```
 """.
 -doc(#{since => <<"OTP R15B01">>}).
+-spec delete(Handle, Dn) -> ok | {ok, Value} |{error, Reason} when
+    Handle :: handle(),
+    Dn :: string(),
+    Value :: term(),
```

Comment:
According to the old doc this function can also return "{ok, {referral, referrals()}}",
which 'Value" must reflect.

Reply:
fixed